pkgs/applications/misc/ecm-tools/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit }:
+stdenv.mkDerivation rec {
+  name = "ecm-tools";
+  version = "master";
```

Comment:
The name should consists of a name part and a version part. Please see the Nixpkgs manual on the naming guidelines.